contracts/PrecedenceCampaignArbitrable.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity ^0.5.8;
+
+import "@aragon/court/contracts/arbitration/IArbitrable.sol";
+import "@aragon/court/contracts/arbitration/IArbitrator.sol";
+
+
+contract PrecedenceCampaignArbitrable is IArbitrable {
+    //bytes4 public constant ERC165_INTERFACE = ERC165_INTERFACE_ID;
+    //bytes4 public constant ARBITRABLE_INTERFACE = ARBITRABLE_INTERFACE_ID;
+    string public constant ERROR_NOT_ALLOWED = "PCA_NOT_ALLOWED";
+
+    address public owner;
+    IArbitrator public arbitrator;
+
+    modifier only(address _who) {
+        require(msg.sender == _who, ERROR_NOT_ALLOWED);
+        _;
+    }
+
+    constructor (address _owner, IArbitrator _arbitrator) public {
+        owner = _owner;
+        arbitrator = _arbitrator;
+    }
+
+    function createDispute(uint8 _possibleRulings, bytes calldata _metadata) external only(owner) {
```

Comment:
Curious on the `uint8`; I was looking around but I think `IArbitrator` accepts uint256 but our implementation only accepts `uint8`?

Reply:
Oh, good point! The [entrypoint accepts `uint256`](https://github.com/aragon/aragon-court/blob/master/contracts/court/AragonCourt.sol#L87)

Reply:
But would things fail if we used a value > max_uint8?

Reply:
Yes, it will revert, there's a `toUint8()` function. We don't expect needing more than 256 options for a dispute. Actually, we expect it to be binary.